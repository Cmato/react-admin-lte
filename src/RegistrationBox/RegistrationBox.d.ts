import * as React from 'react';

export interface RegistrationBoxProps {
}

declare const RegistrationBox: React.ComponentType<RegistrationBoxProps>;

export default RegistrationBox;
