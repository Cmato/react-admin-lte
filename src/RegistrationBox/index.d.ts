export { default } from './RegistrationBox';
export * from './RegistrationBox';
