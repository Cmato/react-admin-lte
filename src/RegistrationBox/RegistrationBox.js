import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

const RegistrationBox = ({
  children,
  className: classNameProp,
  ...other
}) => {
  const className = classNames({
    'register-box': true
  }, classNameProp);

  return (
    <div
      className={className}
      style={{
        marginTop: '0',
        marginBottom: '0',
        paddingTop: '26.5vh',
        paddingBottom: '26.5vh'
      }}
      {...other}
    >
      {children}
    </div>
  );
};

RegistrationBox.propTypes = {
  /**
   * The contents of the component.
   */
  children: PropTypes.node.isRequired,
  /**
   * @ignore
   */
  className: PropTypes.string
};

export default RegistrationBox;
