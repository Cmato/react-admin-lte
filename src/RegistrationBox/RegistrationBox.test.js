import React from 'react';
import { shallow } from 'enzyme';
import RegistrationBox from './RegistrationBox';

it('renders correctly', () => {
  expect(shallow(<RegistrationBox><div /></RegistrationBox>))
    .toMatchSnapshot();
});
