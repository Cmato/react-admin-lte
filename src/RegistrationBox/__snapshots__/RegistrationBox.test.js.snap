// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="register-box"
  style={
    Object {
      "marginBottom": "0",
      "marginTop": "0",
      "paddingBottom": "26.5vh",
      "paddingTop": "26.5vh",
    }
  }
>
  <div />
</div>
`;
